.github/workflows/wheel_linux_aarch64_cuda.yml

```
@@ -90,14 +84,37 @@ jobs:
 
           CIBW_MANYLINUX_AARCH64_IMAGE: manylinux_2_28
 
-          CIBW_BUILD_VERBOSITY: 3
+          CIBW_BUILD_VERBOSITY: 1
```

Could you revert it to 3?
```suggestion
          CIBW_BUILD_VERBOSITY: 3
```